src/Phansible/Controller/BundleController.php

```diff
@@ -124,4 +123,31 @@ protected function outputBundle($zipPath, Application $app, $filename)
 
         return $response;
     }
+
+    /**
+     * @return \Phansible\Model\VagrantBundle
+     */
+    private function getVagrantBundle()
```

Comment:
As flint is silex with cute things.. should be pretty easy.

But I don't really know. So let it as is and I'll check that for another pr